android-project/app/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -1422,6 +1422,7 @@ public void run() {
             mTextEdit.requestFocus();
 
             InputMethodManager imm = (InputMethodManager) SDL.getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+            imm.restartInput(mTextEdit);
```

Comment:
This works, but it shouldn't be called every time. Restarting text input might cause UX problems. Maybe only call it if `mTextEdit` input type is changed.